terraform/deployments/cdn-analytics/bigquery.tf

```diff
@@ -0,0 +1,128 @@
+resource "google_bigquery_dataset" "fastly_logs" {
+  dataset_id = "fastly_logs"
+
+  location = "europe-west2"
+
+  access {
+    role           = "roles/bigquery.admin"
+    group_by_email = "govuk-gcp-access@digital.cabinet-office.gov.uk"
+  }
+
+  access {
+    role           = "roles/bigquery.admin"
+    group_by_email = "analytics-team@digital.cabinet-office.gov.uk"
+  }
+
+  access {
+    role       = "roles/bigquery.dataEditor"
+    iam_member = "serviceAccount:${google_service_account.fastly_writer.email}"
+  }
+}
+
+resource "google_bigquery_table" "fastly_logs" {
+  dataset_id = google_bigquery_dataset.fastly_logs.dataset_id
+  table_id   = "fastly_logs"
+
```

Comment:
can we add time partitioning please, and expire the partitions after 7 days (https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table#nested_time_partitioning)

time_partitioning {
  type  = "DAY"
  expiration_ms = 604800000
}